WebRoot/js/zimbraMail/share/model/ZmInvite.js

```diff
@@ -295,8 +295,8 @@ function(compNum) {
 ZmInvite.prototype.getOrganizerName = 
 function(compNum) {
 	var cn = compNum || 0;
-	return (this.components[cn] && this.components[cn].or)
-		? (this.components[cn].or.d || this.components[cn].or.url) : null;
+	return AjxStringUtil.stripTags((this.components[cn] && this.components[cn].or)
```

Comment:
Can we have the value in a variable then call the stripTags on that?  Doing so much in a single line (condition check, value assignment & passing through utility function) makes it difficult to understand for anyone going through or debugging.